### ISSUE TITLE
twisted.words.protocols.irc.IRCClient.mode docstring claims "#" will be prepended to channel name if it is not already there, but it will not be

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3895]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#3895|
|Type|defect|
|Created|2009-06-29 15:50:28Z|

This part of the docstring should just be deleted, I think.  As far as I can tell, this was never the behavior of this API.

Attachments:

* [words-protocols-irc-IRCClient-mode.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/459/459b865f5d4a076b4ac7e2d44e59bdf71e4bce30/da7132abf2b6e0956928106250847d9dd299c67f.patch) (658 bytes) - added by jesstess on 2009-08-13 18:48:33Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3895 3895
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__words words
keywords__documentation_easy documentation easy
time__1246290628000000 1246290628000000
changetime__1250461735000000 1250461735000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="ivank's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|ivank commented|
|-|-|

I looked very closely at irc.py. The patch looks correct. Someone with SVN access should merge. I'm leaving the review keyword so that this doesn't get lost.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @exarkun|
|-|-|

Thanks.  You should probably remove the review keyword for this sort of thing in the future, or at least assign the ticket to a committer.  Also, would you like commit access?

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [27231]) Apply words-protocols-irc-IRCClient-mode.patch fixing misinformation in `IRCClient.mode` docstring

Author: jesstess
Reviewer: ivank
Fixes: [#3895](https://github.com/twisted/twisted/issues/3895)

Remove the lies in `IRCClient.mode`'s docstring which claimed non-channel targets
would be forced to be channels.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
